build/configs/artik055s/audio/defconfig

```diff
@@ -865,7 +865,11 @@ CONFIG_AUDIO_EXCLUDE_REWIND=y
 #
 CONFIG_MEDIA=y
 CONFIG_MEDIA_PLAYER=y
+CONFIG_MEDIA_PLAYER_STACKSIZE=4096
```

Comment:
I wonder, for Real time oriented applications, do we really need such a big stack ?  In my opinion, its always good to assign less stack and increase it only if necessary.  
Any experimental results can be used to know the real usage of a stack by media threads.

Reply:
Opus takes bunch of stack.